src/main/java/frc/robot/Robot.java

```diff
@@ -118,6 +119,7 @@ public void autonomousPeriodic()
   @Override
   public void teleopInit()
   {
+    m_robotContainer.initialize();
```

Comment:
You probably do not want 2 of these `m_robotContainer.initialize();` commands. I would just delete this one and keep the one below the auto cancel

Reply:
I found it, thanks for pointing it out